src/utils/components/StartPauseModal/StartPauseModal.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import produce from 'immer';
+
+import { ensurePath } from '@catalog/utils/WizardVMContext';
+import { V1VirtualMachine, V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import TabModal from '@kubevirt-utils/components/TabModal/TabModal';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Checkbox, Form, FormGroup } from '@patternfly/react-core';
+
+import { ModalPendingChangesAlert } from '../PendingChanges/ModalPendingChangesAlert/ModalPendingChangesAlert';
+import { getChangedStartStrategy } from '../PendingChanges/utils/helpers';
+
+type StartPauseModalProps = {
+  vm: V1VirtualMachine;
+  isOpen: boolean;
+  onClose: () => void;
+  onSubmit: (updatedVM: V1VirtualMachine) => Promise<V1VirtualMachine | void>;
+  headerText: string;
+  vmi?: V1VirtualMachineInstance;
+};
+
+const StartPauseModal: React.FC<StartPauseModalProps> = ({
+  vm,
+  isOpen,
+  onClose,
+  onSubmit,
+  headerText,
+  vmi,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [checked, setChecked] = React.useState<boolean>(!!vm?.spec?.template?.spec?.startStrategy);
+
+  const updatedVirtualMachine = React.useMemo(() => {
+    const updatedVM = produce<V1VirtualMachine>(vm, (vmDraft: V1VirtualMachine) => {
+      ensurePath(vmDraft, ['spec.template.spec']);
+      vmDraft.spec.template.spec.startStrategy = checked ? 'Paused' : null;
+    });
+    return updatedVM;
+  }, [vm, checked]);
+  return (
+    <TabModal
+      obj={updatedVirtualMachine}
+      isOpen={isOpen}
+      onClose={onClose}
+      onSubmit={onSubmit}
+      headerText={headerText}
+    >
+      <Form>
+        {vmi && (
+          <ModalPendingChangesAlert
+            isChanged={getChangedStartStrategy(updatedVirtualMachine, vmi, checked)}
+          />
+        )}
+        <FormGroup
+          fieldId="start-pause-mode"
+          helperText={t(
+            'Applying the Start/Pause mode to this Virtual Machine will cause it to partially reboot and pause.',
```

Comment:
please change to `Applying the start/pause mode to this VirtualMachine will cause it to partially reboot and pause.`